src/applause-button.scss

```diff
@@ -43,15 +42,37 @@ applause-button {
     opacity: 0.5;
   }
 
-  .shockwave {
+  .applaud-button {
     @include position-fill();
+    cursor: pointer;
+    display: block;
     border-radius: 50%;
+    border-color: inherit;
+    border: 1px solid;
+    width: 100%;
+    height: 100%;
+    background-color: transparent;
+    padding: 0;
+    color: inherit;
+    outline: none;
+    &:focus, &:hover {
+      box-shadow: 0 0 10px;
+      opacity: 1;
+    }
+  }
+
+  &:not(.clapped) .applaud-button:focus, &:not(.clapped) .applaud-button:hover {
+    animation-name: shockwave;
+    animation-duration: 1s;
+    animation-iteration-count: infinite;
```

Comment:
For the record, I strongly dislike the infinite pulsing. At the very least, due to accessibility concerns, it should be put within a media query:

```
@media (prefers-reduced-motion: no-preference) {
  &:not(.clapped) .applaud-button:focus, &:not(.clapped) .applaud-button:hover {
    animation-iteration-count: infinite;
  }
}
```

The fallback behaviour would then be a once-only animation.